htdocs/compta/bank/class/paymentvarious.class.php

```diff
@@ -125,6 +125,16 @@ class PaymentVarious extends CommonObject
 	 */
 	public $fk_user_modif;
 
+	/**
+	 * @var int ID
+	 */
+	public $user_modif;
+
+	/**
+	 * @var int timestamp
+	 */
+	public $date_modif;
```

Comment:
We already have 
$date_modification
into commonobject. We should instead rename date_modification into date_modif